File: docs/source/examples/Query/String.java
RealmQuery<Project> projectsQuery = realm.where(Project.class);

// Pass Case.INSENSITIVE as the third argument for case insensitivity.
Log.i(TAG, "Projects that start with 'e': "
  + projectsQuery.beginsWith("name", "e", Case.INSENSITIVE).count());

Log.i(TAG, "Projects that contain 'ie': "
  + projectsQuery.contains("name", "ie").count());

File: docs/source/examples/Query/String.js
// Use [c] for case-insensitivity.
console.log(
  "Projects that start with 'e': " +
    projects.filtered("name BEGINSWITH[c] 'e'").length
);
console.log(
  "Projects that contain 'ie': " +
    projects.filtered("name CONTAINS 'ie'").length
);